doc/user/content/sql/alter-owner.md

```diff
@@ -9,9 +9,8 @@ menu:
 `ALTER ... OWNER` updates the owner of an item.
 
 {{< warning >}}
-Roles in Materialize are currently limited in functionality. In the future they
-will be used for role-based access control. See GitHub issue {{% gh 11579 %}}
-for details.
+Role-based access control (RBAC) is disabled by default. Please reach out to support if you would
+like to enable RBAC.
 {{< /warning >}}
```

Comment:
Do we consider any feature behind a feature flag to be in alpha? For consistency, I think these should all use the alpha annotation (like [here](https://materialize.com/docs/sql/create-connection/#aws-privatelink)):

```
{{< alpha />}}
```

@tr0njavolta, one improvement here could be to allow the [shortcode](https://github.com/MaterializeInc/materialize/blob/main/doc/user/layouts/shortcodes/alpha.html) to take the name of the feature as an argument. This would make it more flexible as we release more features behind a feature flag.